docs/starters.yml

```
@@ -5066,6 +5066,26 @@
     - SEO and Open graphs support
     - Color modes
     - Code Highlighting
+- url: https://london-night-day.netlify.com/
+  repo: https://github.com/jooplaan/gatsby-london-night-and-day
+  description: A custom, image-centric dark and light mode aware theme for Gatsby. Advanced from the Gatsby starter London After Midnight.
+  tags:
+    - Blog
+    - Portfolio
+    - Gallery
+    - SEO
+    - Markdown
+    - Sass/SCSS    
```

```suggestion
    - Styling:SCSS    
```
In order for tests to pass it needs to match the specific tag.